.github/workflows/tx-pull.yml

```diff
@@ -46,33 +48,10 @@ jobs:
         # See https://git-scm.com/docs/git-add#_options
         add_options: '-A'
 
-        # Optional. Options used by `git-push`.
-        # See https://git-scm.com/docs/git-push#_options
-        push_options: '--force'
-
         # Optional. Disable dirty check and always try to create a commit and push
-        skip_dirty_check: true
-
-        # Optional. Skip internal call to `git fetch`
-        skip_fetch: true
+        skip_dirty_check: false
 
-        # Optional. Skip internal call to `git checkout`
-        skip_checkout: true
```

Comment:
keeping default checks

Reply:
when tx-pull updates the tracked files, are there dirty files we don't want to commit?

Reply:
I didn't find any, I think those were enabled to have dirty files on the branch that gets the translations.

Reply:
i'm not sure I understand your response. IIRC, When I set up this workflow, there is a chance of dirty files that were not related to translations, and we shouldn't care about anything except the non `en` language files that are updated by transifex, and we should accept those as transifex is the source of truth for non `en` language files.